mmv1/templates/terraform/resource.html.markdown.erb

```diff
@@ -193,6 +193,27 @@ This resource does not support import.
 
 <%= object.name -%> can be imported using any of these accepted formats:
 
+```
+<% import_id_formats_from_resource(object).map{ |id| id.gsub('%', '') }.each do |id_format| -%>
+<%= id_format %>
+<% end -%>
+```
+
+In Terraform v1.5.0 and later, use an [`import` block](https://developer.hashicorp.com/terraform/language/import) to import <%= object.name -%> using one of the formats above. For example:
```

Comment:
I'm not super familiar, but this wording implies that this method is preferred. Is that the case? If not, we may want to loosen to something like `you can use an import ...`. Part of my thinking is that existing users may come to this block looking for import statements because they are already familiar with them.

Reply:
That seems fair, thanks for explaining 👍